x/farming/spec/07_params.md

```diff
@@ -20,4 +21,11 @@ Fee paid for to create a Private type Farming plan. This fee prevents spamming a
 
 ## FarmingFeeCollector
 
-A farming fee collector is a module account address that collects farming fees, such as staking creation fee and private plan creation fee.
\ No newline at end of file
+A farming fee collector is a module account address that collects farming fees, such as staking creation fee and private plan creation fee.
+
+## DelayedStakingGasFee
+
+Since the farming module has adopted delayed staking,
+handling `MsgStake` does not immediately consumes gas.
+Instead, gas for this newly queued staking is consumed at the end of the epoch.
+`DelayedStakingGasFee` is to impose gas fee for this reason.
```

Comment:
Could you describe it in more detail?  We can commit on https://github.com/tendermint/farming/pull/178 continuously @hallazzang